server/backend/TextToSpeech/YngpingAudioSynthesizer.cs

```diff
@@ -57,34 +57,31 @@ private static byte[] TrimSilence(string[] inputList, int msReserved = 0)
                 int startBytes = 0, endBytes = 0, startPos = 0, endPos = 0;
                 foreach (string inputPath in inputList)
                 {
-
                     using (WaveFileReader reader = new WaveFileReader(inputPath))
                     {
                         // int bytesPerSample = reader.WaveFormat.BitsPerSample / 8;
                         // Assuming bytesPerSample == 2
                         int bytesPerMillisecond = reader.WaveFormat.AverageBytesPerSecond / 1000;
-
                         int bytesRead = reader.Read(buffer, 0, bytesPerMillisecond);
```

Comment:
I wonder if there is any point of reading the file in 1 ms chunks. Should be fine to just read the whole file.

Reply:
Hmmmmm I've never thought of that... I was just thinking since we are cutting it with 1ms granularity, we could read it by 1ms chunk.